open-source.php

```diff
@@ -155,6 +155,38 @@
     </div>
 </div>
 
+<div class="grid">
+    <div class="whole">
+        <div class="two-thirds">
+            <h2>Projects We've Supported</h2>
+            <p>elementary makes a point to send funds back to the projects we rely on. When you purchase a copy of elementary OS, you're also supporting great projects like these.</p>
+        </div>
+    </div>
+    <div class="donation-grid">
+        <a href="https://salt.bountysource.com/teams/bountysource" target="_blank">
```

Comment:
You shouldn't need to `target="_blank"` these links, the website does that automatically.